DOCS/man/options.rst

```diff
@@ -3064,6 +3064,14 @@ Window
 ``--snap-window``
     (Windows only) Snap the player window to screen edges.
 
+``--drag-and-drop=<auto|replace|append>``
+    (X11 and Wayland only)
+    Controls the default behavior of drag and drop on platforms that support this.
+    ``auto`` will obey what the underlying os/platform gives mpv. Typically, holding
+    shift during the drag and drop will append the item to the playlist. Otherwise,
+    it will completely replace it. ``replace`` and ``append`` always force replacing
+    appending to the playlist respectively.
```

Comment:
missing "and" between `replacing appending`.

Reply:
Docs are hard.